xgcm/test/test_metric_ops_multi_axis.py

```diff
@@ -50,12 +51,12 @@ def test_bgrid(self, funcname, boundary, periodic):
             expected = _expected_result(
                 ds.tracer, ds[metric_name], grid, dim, axis, funcname, **kwargs
             )
-            assert new.equals(expected)
+            xr.testing.assert_allclose(new, expected)
```

Comment:
shall we do `from xarray.testing import assert_allclose` at the top?

Reply:
Done